python_src/src/lamp_py/performance_manager/l1_rt_trips.py

```diff
@@ -284,7 +284,10 @@ def update_start_times(db_manager: DatabaseManager) -> None:
     unscheduled_start_times = db_manager.select_as_dataframe(
         sa.select(
             VehicleEvents.pm_trip_id.label("b_pm_trip_id"),
-            sa.func.min(VehicleEvents.vp_move_timestamp).label("b_start_time"),
+            sa.func.coalesce(
+                sa.func.min(VehicleEvents.vp_move_timestamp),
+                sa.func.min(VehicleEvents.vp_stop_timestamp),
+            ).label("b_start_time"),
```

Comment:
there were edge cases where there were no move timestamps and it would cause an error.